src/js/components/TaskListItemComponent.jsx

```diff
@@ -190,6 +190,13 @@ var TaskListItemComponent = React.createClass({
     </a>);
   },
 
+  getHostCpuListLink: function () {
+    const Link = Config.hostCpuListLinkGenerator();
```

Comment:
FAR: it would be great to pass the app/task as parameters, to highlight the app/task in the displayed table